tests/e2e/tests/checkout/order-confirmation.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,311 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	FREE_SHIPPING_NAME,
+	FREE_SHIPPING_PRICE,
+	SIMPLE_PHYSICAL_PRODUCT_NAME,
+	SIMPLE_VIRTUAL_PRODUCT_NAME,
+} from './constants';
+import { CheckoutPage } from './checkout.page';
+
+const testData = {
+	firstname: 'John',
+	lastname: 'Doe',
+	addressfirstline: '123 Easy Street',
+	addresssecondline: 'Testville',
+	country: 'United States (US)',
+	city: 'New York',
+	state: 'New York',
+	postcode: '90210',
+	email: 'john.doe@test.com',
+	phone: '01234567890',
+};
+
+const test = base.extend< { pageObject: CheckoutPage } >( {
+	pageObject: async ( { page }, use ) => {
+		const pageObject = new CheckoutPage( {
+			page,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( 'Shopper → Order Confirmation', () => {
+	test.beforeEach( async ( { admin, editorUtils, frontendUtils } ) => {
+		await frontendUtils.login( true );
+		await admin.visitSiteEditor( {
+			postId: 'woocommerce/woocommerce//order-confirmation',
+			postType: 'wp_template',
+		} );
+		await editorUtils.enterEditMode();
+		await editorUtils.closeWelcomeGuideModal();
+		await editorUtils.transformIntoBlocks();
+	} );
+
+	test.afterAll( async ( { frontendUtils } ) => {
+		await frontendUtils.login( true );
+	} );
+
+	test( 'Place order as a logged in user', async ( {
+		frontendUtils,
+		pageObject,
+		page,
+		browser,
+	} ) => {
+		await frontendUtils.emptyCart();
+		await frontendUtils.goToShop();
+		await frontendUtils.addToCart( SIMPLE_PHYSICAL_PRODUCT_NAME );
+		await frontendUtils.addToCart( SIMPLE_VIRTUAL_PRODUCT_NAME );
+		await frontendUtils.goToCheckout();
+		await pageObject.page
```

Comment:
Could try `selectAndVerifyShippingOption` here instead. I don't know why it fails though.